os/board/rtl8721csm/src/component/os/tizenrt/rtk_netmgr.c

```diff
@@ -382,9 +382,16 @@ trwifi_result_e wifi_netmgr_utils_deinit(struct netdev *dev)
 trwifi_result_e wifi_netmgr_utils_scan_ap(struct netdev *dev, trwifi_ap_config_s *config)
 {
 	trwifi_result_e wuret = TRWIFI_FAIL;
-	if(wifi_scan_networks(app_scan_result_handler, NULL ) != RTW_SUCCESS){
-		//ndbg("[RTK] [ERR] WiFi scan fail(%d)\n", ret);
-		return wuret;
+	if (config->ssid != NULL) {
+		int scan_buf_len = 50;
+		if (wifi_scan_networks_with_ssid(NULL, NULL, scan_buf_len, config->ssid, config->ssid_length) != RTW_SUCCESS) {
+			return wuret;

```

Comment:
`wuret` is not set from the return of functions.
So I think this variable is not needed.
How about using like `return TRWIFI_FAIL` or `return `TRWIFI_SUCCESS`?